submissions/5mountains/dom-api-task/js/main.js

```diff
@@ -0,0 +1,102 @@
+const styles = {
+    menuLink: 'menu__link',
+    menuItem: 'menu__item',
+    heroItem: 'hero__item',
+    heroImg: 'hero__image',
+    itemIcon: 'item__icon',
+    itemTitle: 'item__caption',
+    itemPrice: 'item__price',
+    itemInfo: 'item__description'
+};
+
+async function fetchAsync () {
+    let response = await fetch('./data/data.json');
```

Comment:
Use `const` instead of `let` in these cases. And change it in other places

Reply:
done